### DISCUSSION TITLE
ZeroQL.CLI is not build with .NET 8

### DISCUSSION THREAD

**Discussion by quyvu01** [byme8/ZeroQL#89] [Q&A] [answered]
Hi, I'm using .NET 8 and I created some images, which include ZeroQL library. I already added the dotnet tool Install ZeroQL.CLI to the Dockerfile and it seems this one is not available for .NET8. Can you please upgrade for it?

**Comment by mathiash98** [comment, marked as answer]
Try out https://github.com/byme8/ZeroQL/releases/tag/v6.2.0-preview.1 which should have initial support for Dotnet 8

**Reply by quyvu01** [reply]
Thank you so much!
